src/index.js

```diff
@@ -149,9 +149,14 @@ export function createComplexityLimitRule(maxCost, options = {}) {
         }
 
         if (node.kind === 'Document') {
-          if (visitor.getCost() > maxCost) {
+          const cost = visitor.getCost();
+          if (options.onCost) options.onCost(cost);
+          if (cost > maxCost) {
+            const errorMessage = options.formatErrorMessage ?
+              options.formatErrorMessage(cost) :
+              `query exceeds complexity limit. Cost: ${cost}`;
```

Comment:
we should absolutely _not_ always return the cost here; the calculation may be a private detail